internal/app/go-filecoin/porcelain/miner.go

```diff
@@ -208,6 +211,14 @@ func MinerGetStatus(ctx context.Context, plumbing minerStatusPlumbing, minerAddr
 	if err != nil {
 		return MinerStatus{}, err
 	}
+	sectorCount := 0
```

Comment:
I think sector count is a intrinsically useful statistic, but it's the only artifact on chain that indicates sealing has been successful. It's used in the TestMiningSealSector test.